source/common/upstream/outlier_detection_impl.cc

```diff
@@ -207,10 +233,61 @@ void DetectorImpl::onConsecutive5xxWorker(HostPtr host) {
   ejectHost(host, EjectionType::Consecutive5xx);
 }
 
+const double Utility::SR_STDEV_FACTOR = 1.9;
+
+double Utility::srEjectionThreshold(double sr_sum, std::vector<double>& sr_data) {
+  double mean = sr_sum / sr_data.size();
```

Comment:
Add comment in this function on what all of this math is actually doing, with an example.